lib/streamlit/DeltaGenerator.py

```diff
@@ -773,9 +781,20 @@ def bar_chart(self, element, data, width=0, height=0):
             height: 200px
 
         """
-        from streamlit.elements.Chart import Chart
-        chart = Chart(data, type='bar_chart', width=width, height=height)
-        chart.marshall(element.chart)
+        import altair as alt
```

Comment:
Import pandas here, not at the top.

We try to import things as close as possible to where they are used, to reduce Streamlit's load time.

So if someone is using Streamlit but not doing anything that requires DataFrames, their load time will be faster.